source/material_model/compositing.cc

```diff
@@ -246,7 +246,12 @@ namespace aspect
                                    "compositing",
                                    "The ``compositing'' Material model selects material model properties from a "
                                    "given set of other material models, and is intended to make mixing different "
-                                   "material models easier."
+                                   "material models easier. This is useful, for example, when wanting to "
+                                   "use the melting parameterization of the `melt simple' model (which has "
```

Comment:
In the sentence above, two ticks are used for the name of the compositing material model (``compositing''). I don't know whether we use one or two ticks, but I guess it should be the same everywhere.